### DISCUSSION TITLE
Task 1 - Plotly graphs

### DISCUSSION THREAD

**Discussion by n-leadbetter** [fusion-energy/neutronics-workshop#247] [Q&A]
Hi there!

Firstly thank you for making this resource.

I am new to OpenMC and have very little coding experience prior to this so please excuse if I make any obvious mistakes.

I have been running through Task 1 as written. I am aware that the accompanying videos are out of date, but I was wondering how would I obtain a customisable graph from plotly like in these videos rather than a static image?

Thanks in advance!
Nathan

**Comment by shimwell** [comment]
I do miss those nice plotly plots. We improved the openmc plot xs method so that it was super flexible and would allow different isotopes,  reactions and I moved to use that method instead as it simplified things for the user a lot.

The source code is here, so one could write a plotly version of this but I don't see that getting merged in to openmc as plotly is not in the openmc dependencies

Perhaps have a go at adapting this method 
https://github.com/openmc-dev/openmc/blob/751ea980749bf754516caf4574e3295e1752a56f/openmc/plotter.py#L100

**Comment by shimwell** [comment]
I should have mentioned I have a few other isotope and material cross section plotting webpages that can be used to show plotly graphs of cross sections

https://xsplot.com/

**Reply by n-leadbetter** [reply]
Thanks for letting me know!
